test/e2e/framework/accessory.go

```diff
@@ -93,7 +93,7 @@ func (a *Accessory) Run(t *testing.T, opts ...RunOption) error {
 		return err
 	}
 	go func() {
-		defer func() { cleanupCancel() }()
+		defer cleanupCancel()
```

Comment:
Does this call `cleanupCancel()` at the time the `defer` is registered or at the time it's fired?